app/controllers/responses_controller.rb

```
@@ -0,0 +1,249 @@
+  private
+
+  def set_response
+    @response = Response.find_by(id: params[:id])
+  end
+
+  def response_params
+    params.require(:response).permit(
+      :map_id,
+      :is_submitted,
+      :submitted_at,
+      scores_attributes: [:id, :item_id, :answer, :comments]
```

RuboCop (1.86.1)

[convention] Use `%i` or `%I` for an array of symbols.

(Style/SymbolArray)